rust/MEMORY_PRESSURE_FIXES.md

```
@@ -0,0 +1,205 @@
+### Check Current Memory Usage:
+```bash
+top -l 1 | grep "PhysMem"
+pkillall -HUP top
+```
+
+### Check MLX Memory Limits:
+```bash
+your_ai train --model llama-8b --max-memory 70.0
+```
+Look for output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)